bot/exts/moderation/alts.py

```diff
@@ -0,0 +1,169 @@
+import gettext
+
+import discord
+from discord.ext import commands
+from pydis_core.site_api import ResponseCodeError
+from pydis_core.utils.members import get_or_fetch_member
+
+from bot import constants
+from bot.bot import Bot
+from bot.converters import UnambiguousMemberOrUser
+from bot.log import get_logger
+from bot.pagination import LinePaginator
+from bot.utils.channel import is_mod_channel
+from bot.utils.time import discord_timestamp
+
+log = get_logger(__name__)
+
+
+class AlternateAccounts(commands.Cog):
+    """A cog used to track a user's alternative accounts across Discord."""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+
+    @staticmethod
+    def error_text_from_error(error: ResponseCodeError) -> str:
```

Comment:
Wouldn't it be better to handle each response code gracefully for each endpoint ?

Reply:
There's overlapping between what errors can be returned by what endpoint, so this was the easiest, and most extensible way i could think of doing this